nodes/src/nodes/agent_crewai/crewai.py

```
@@ -285,20 +347,215 @@ def _on_any_event(source, event):
+
+        descriptors = []
+        for node_id in crewai_node_ids:
+            req = IInvokeCrew.Describe()
             try:
-                final_text = _safe_str(getattr(result, 'raw'))
+                pSelf.instance.invoke('crewai', req, nodeId=node_id)
             except Exception:
-                final_text = ''
-        if not final_text:
-            final_text = _safe_str(result)
+                pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)